recipes/angsd/build.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+binaries="\
+misc/msToGlf \
+misc/contamination \
+misc/contamination2 \
+misc/splitgl \
+misc/thetaStat \
+misc/printIcounts \
+misc/NGSadmix \
+misc/realSFS \
+misc/smartCount \
+misc/supersim \
+angsd \
+"
+BINDIR=$PREFIX/bin
+mkdir -p $BINDIR
+
+
+make LDFLAGS="-L=${PREFIX}/lib" INC="-I=${PREFIX}/include"
```

Comment:
Done; I wasn't aware of these variables, and what worked in the end was CPLUS_INCLUDE_PATH and C_INCLUDE_PATH.